api/rest/client/methods.go

```diff
@@ -77,9 +77,9 @@ func (c *Client) Unpin(ci *cid.Cid) error {
 
 // Allocations returns the consensus state listing all tracked items and
 // the peers that should be pinning them.
-func (c *Client) Allocations(pinType api.FilterType) ([]api.Pin, error) {
+func (c *Client) Allocations(pinType string) ([]api.Pin, error) {
```

Comment:
`type types.PinType`. Receive a PinType, send a string on the query (because that's user friendly)